src/apps/search-header/search-header.dev.tsx

```diff
@@ -1,13 +1,14 @@
 import { ComponentMeta, ComponentStory } from "@storybook/react";
 import * as React from "react";
 import StoryHeader from "../../components/search-bar/story-header.dev.inc";
-import SearchHeader, { SearchHeaderProps } from "./search-header";
+import { SearchHeaderProps } from "./search-header";
+import SearchHeaderEntry from "./search-header.entry";
 
 export default {
   title: "Apps / Search Header",
-  component: SearchHeader,
+  component: SearchHeaderEntry,
```

Comment:
Yes :) We show the entry file in the app story because we want to be able to show all the props that are defined on the app. I followed Mikkel's example for Hello world and search header demo apps.